packages/bpk-component-price-range/src/BpkPriceMarker.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { ForwardedRef } from 'react';
+import { forwardRef } from 'react';
+
+import BpkText, { TEXT_STYLES } from '../../bpk-component-text/src/BpkText';
+import { cssModules } from '../../bpk-react-utils';
+
+import type { MarkerType } from './common-types';
+
+import STYLES from './BpkPriceMarker.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+type Props = {
+  priceLabel: string;
+  type: MarkerType;
+};
+
+const BpkPriceMarker = (
+  { priceLabel, type }: Props,
+  ref: ForwardedRef<HTMLDivElement>,
```

Comment:
the same here, so that you don't need suppress the typescript complaint in [test](https://github.com/Skyscanner/backpack/pull/3773/files#diff-2d0da3a6178c173f9989b54b3132729aade455c5c5d3a074d07cdec5a61cbd21R79)

Reply:
This is a child component and don't expose it, so we don't need to add `rest`